tests/unit/server-entry.spec.ts

```diff
@@ -2,14 +2,56 @@ import serverEntry from '@/server-entry';
 import TermboxRequest from '@/common/TermboxRequest';
 import EntityNotFound from '@/common/data-access/error/EntityNotFound';
 import BundleBoundaryPassingException, { ErrorReason } from '@/server/exceptions/BundleBoundaryPassingException';
+import BundleRendererContext from '@/server/bundle-renderer/BundleRendererContext';
+import BundleRendererServices from '@/server/bundle-renderer/BundleRendererServices';
 
 const mockBuildApp = jest.fn();
 jest.mock( '@/common/buildApp', () => ( {
 	__esModule: true,
 	default: ( termboxRequest: TermboxRequest ) => mockBuildApp( termboxRequest ),
 } ) );
 
+const getMediawikiBot = jest.fn();
+function getMockBundleRendererServices() {
+	const services = {} as BundleRendererServices;
+	Object.defineProperty( services, 'mediawikiBot', {
```

Comment:
I tried and failed to get rid of this while keeping the spy on the property access :(
I found out that it is possible to mock/spy on getters which is nice. Doesn't help in this case, though.

Maybe an alternative would have been to assert that the same mwbot instance was passed to the constructors of LanguagesRepo and EntityRepo instead. OTOH I don't think it's worth the hassle to change this now.